config_amboy.go

```diff
@@ -40,8 +40,10 @@ type AmboyConfig struct {
 
 // AmboyDBConfig configures Amboy's database connection.
 type AmboyDBConfig struct {
-	URL      string `bson:"url" json:"url" yaml:"url"`
-	Database string `bson:"database" json:"database" yaml:"database"`
+	URL string `bson:"url" json:"url" yaml:"url"`
+	// TODO (DEVPROD-4163): remove KanopyURL after the cutover to Kanopy.
+	KanopyURL string `bson:"kanopy_url" json:"kanopy_url" yaml:"kanopy_url"`
```

Comment:
I think you need to set this field below in `Set` (or else it won't get set from the admin page).

Reply:
It seems that [this field](https://github.com/evergreen-ci/evergreen/blob/6de90900d38382fee2cda1f1d2ba8cfd4adceb1b/config_amboy.go#L112) sets the value to whatever is in the `DBConnection` struct.
It worked for me in local-evergreen, fwiw.

Reply:
Ah I see, that makes sense.